pages/pipelines/command_step.md

```diff
@@ -302,8 +302,35 @@ _Optional Attributes_
     <td><code>exit_status</code></td>
     <td>
       The exit status number that will cause this job to retry ('*' does not include 0) <br>
-      <em>Example:</em> <code>"*"</code><br>
-      <em>Example:</em> <code>2</code>
+      <p><em>Examples:</em></p>
+      <ul>
+        <li><code>"*"</code></li>
+        <li><code>2</code></li>
+      </ul>
+    </td>
+  </tr>
+  <tr>
+    <td><code>signal</code></td>
+    <td>
+      The signal that will cause this job to retry. This signal will only be present if the agent sends a signal to the job and an interior process does not handle the signal. `SIGKILL` propagates reliably because it cannot be handled, and is a useful to differentiate graceful cancelation and timeouts.
```

Comment:
Typo + formatting:

```suggestion
      The signal that will cause this job to retry. This signal will only be present if the agent sends a signal to the job and an interior process does not handle the signal. <code>SIGKILL</code> propagates reliably because it cannot be handled, and is useful to differentiate graceful cancelation and timeouts.
```

Or, with my nitpicks:

- Avoid "will"
- Being "present" sounds a bit unclear to me, but I'm not sure if what I've got is right.

```suggestion
      The signal that causes this job to retry. This signal only occurs when the agent sends a signal to the job and an interior process does not handle the signal. <code>SIGKILL</code> propagates reliably because it cannot be handled, and is useful to differentiate graceful cancelation and timeouts.
```

Reply:
Nice. I'm used to ["present" in the rails sense](https://api.rubyonrails.org/classes/Object.html#method-i-present-3F), but that's probably not widely accessible here. "occurs" sounds like the signal might only be _sent_ in limited cases, which isn't quite right. I've had a go with "appears".